README.md

```diff
@@ -61,6 +61,7 @@ Your contributions are always welcome!
 -  [ITL-Java](https://github.com/fikr4n/itl-java) : ITL Java is a Java library based on LGPL-licensed ITL (Islamic Tools and Libraries), currently includes prayer times (salat), Hijri date, and qibla direction.
 - [iclib-java](https://github.com/fikr4n/iclib-java) : Islamic Calculation Library (ICLib) contains calculations for prayer (salat) times, qibla direction, and Hijri conversion
 - [Pray-Times](https://github.com/abodehq/Pray-Times/tree/master/Code/v2/java) : an open-source library for calculating Muslim prayers times.
+- [Android-Open-Hadith-Project](https://github.com/mhashim6/Android-Open-Hadith-Project) : An open Hadith Library for Android that encapsulates databases of 'Sahih Al-Bukhari' and Sahih Muslim.
```

Comment:
sorry, I know, the repos are currently down for some reasons, when I re-upload again I'll inform you.
thanks.